src/handlers/select-answer.js

```diff
@@ -0,0 +1,17 @@
+import { setStatusClass } from '../views/set-status.js';
+/*import { displayQuestion } from "../views/questions-list.js"; */
+
+ export function selectAnswer(event){
+    const selectedButton = event.target;
+    let correct = selectedButton.dataset.correct;
+    setStatusClass(document.body, correct)
+
+    Array.from(document.getElementById('answer-buttons').children).forEach(
```

Comment:
Up to you if you want to change it, but I generally prefer to put the `document...children` bit in a separate line and assign it to a variable. Then you can do `for (const button of [...answerButtons]) {` (using the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax))